pnpm-lock.yaml

```diff
@@ -19480,7 +19477,7 @@ snapshots:
   '@vue/test-utils@2.4.6':
     dependencies:
       js-beautify: 1.14.9
-      vue-component-type-helpers: 2.2.2
```

Comment:
these changes look unrelated

Reply:
I created a new branch from current master, upgraded the SDK and recreated the lockfile, and I'm seeing the Vue changes still. Should I undo those manually?